src/components/views/messages/MFileBody.tsx

```diff
@@ -223,7 +223,15 @@ export default class MFileBody extends React.Component<IProps, IState> {
             </span>;
         }
 
-        const showDownloadLink = this.props.tileShape || !this.props.showGenericPlaceholder;
```

Comment:
the check is required for external dependencies (non-public - DM/ask internally please)